metaflow/decorators.py

```diff
@@ -273,7 +273,8 @@ def task_pre_step(self,
                       graph,
                       retry_count,
                       max_user_code_retries,
-                      ubf_context):
+                      ubf_context,
+                      input_obj):
```

Comment:
Rather than `input_obj`, can we use `inputs` - that seems semantically correct looking at L433 of task.py

Reply:
I can change that. No issues.